contributors.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: page
+title: Contributors
+nav_order: 6
+permalink: /contributors/
+---
+
+# Contributors
+
+Slingshot is a development project in Palladio that enhances the simulation and adds new modelling capabilities to Palladio.
+Slingshot started at the Software Quality and Architecture (SQA) group at the University of Stuttgart.  
```

Comment:
should we also state our institute? like, mention ISTE somewhere? just a thought though. not sure, whether it would be an improvement, or not.